src/main/java/serverutils/ServerUtilities.java

```diff
@@ -134,14 +133,15 @@ public void onServerStarted(FMLServerStartedEvent event) {
         if (Ranks.isActive()) {
             Ranks.INSTANCE.commands.clear();
 
-            boolean crucibleLoaded = Loader.isModLoaded("Crucible");
+            boolean bukkitLoaded = CommonUtils.getClassExists("thermos.ThermosRemapper")
```

Comment:
Another common one is "org.bukkit.World"

Reply:
Would it also make sense to have a cmdfile override to shut the feature off so other examples of these could be manually handled by cfg ? cmdoverride=off kinda thing

Reply:
`-DforceBukkit=true`

Reply:
Bukkit path added aswell.

@bombcar cmd override can already be turned off in config if that's what you mean.